src/providers/exploreState.tsx

```diff
@@ -413,13 +421,9 @@ function exploreReducer(
         ...state,
         categoryGroupConfigs: entityPageState[tabValue].categoryGroupConfigs,
         categoryViews: nextCategoryViews ?? categoryViews,
-        entityPageState: {
-          ...entityPageState,
-          [tabValue]: {
-            ...entityPageState[tabValue],
-            categoryViews: nextCategoryViews ?? categoryViews,
-          },
-        },
+        entityPageState: updateEntityPageState(tabValue, entityPageState, {
```

Comment:
`updateEntityPageState` function to update the entity's `entityPageState` with the next category views.